libs/core/langchain_core/messages/modifier.py

```diff
@@ -0,0 +1,38 @@
+from typing import Any, List, Literal
+
+from langchain_core.messages.base import BaseMessage, BaseMessageChunk
+
+
+class ModifierMessage(BaseMessage):
+    """Message responsible for modifying other messages (deleting / updating.)"""
+
+    def __init__(self, id: str, **kwargs: Any) -> None:
```

Comment:
i considered this -- there is an issue w/ `content` field during serialization/deserialization. since it's required on the base message, we still need to pass it here at deserialization. i can raise for any other keys though!